src/CMakeLists.txt

```diff
@@ -152,13 +180,23 @@ if(MUPENPLUSAPI)
   include_directories( . inc )
   set(GLideN64_SOURCES_UNIX
 	MupenPlusPluginAPI.cpp
-	mupenplus/Config_mupenplus.cpp
 	mupenplus/CommonAPIImpl_mupenplus.cpp
 	mupenplus/MemoryStatus_mupenplus.cpp
 	mupenplus/MupenPlusAPIImpl.cpp
 	Graphics/OpenGLContext/mupen64plus/mupen64plus_DisplayWindow.cpp
   )
 
+  if(MUPENPLUSAPI_GLIDENUI)
+      add_definitions(-DGLIDENUI)
```

Comment:
I would rename this define to something like M64P_GLIDENUI to make it clear it's only related to mupen64plus+GLideNUI

Reply:
Fixed